--- venv/lib/python3.8/site-packages/weasyprint/__init__.py	2021-02-21 14:10:44 +0000
+++ bin/weasyprint/__init__.py	2021-02-21 14:57:52 +0000
@@ -31,7 +31,8 @@
     else:
         # Frozen with something else (py2exe, etc.)
         # See https://github.com/Kozea/WeasyPrint/pull/269
-        ROOT = Path(os.path.dirname(sys.executable))
+        # ROOT = Path(os.path.dirname(sys.executable))
+        ROOT = Path(os.path.dirname(sys.executable)) / "lib" / "weasyprint"
 else:
     ROOT = Path(os.path.dirname(__file__))
